gradle-incremental-configuration-cache/src/main/java/com/palantir/gradle/configcache/incremental/IncrementalConfigurationCachePlugin.java

```diff
@@ -74,6 +76,14 @@ public final void apply(Project project) {
             }
         }));
 
+        TaskProvider<ValidateConfigurationCacheTask> validateAllowList = project.getTasks()
+                .register("validateConfigurationCacheAllowList", ValidateConfigurationCacheTask.class, task -> {
```

Comment:
Lets call it `dryRunConfigurationCacheAllowListTasks` or something similar - this sounds like it is actually checking the content of the allowlist, like a lockfile verification.